requirements/conda/default.yml

```diff
@@ -0,0 +1,5 @@
+dependencies:
+  - python
+  - numpy
+  - scipy
+  - cython
```

Comment:
Cython is only needed for building, not for installing and importing, so I'm not sure it's a good idea to have it in default

Reply:
Good point, I totally agree. I'll remove the `cython` line.